neptune/new/project.py

```diff
@@ -157,3 +157,204 @@ def _as_list(
             return [value]
         verify_collection_type(name, value, str)
         return value
+
+    def assign(self, value, wait: bool = False) -> None:
+        """Assign values to multiple fields from a dictionary.
+        You can use this method to quickly log all projects's parameters.
```

Comment:
->  You can use this method to log multiple pieces of information with one command.